CondFormats/SiStripObjects/src/SiStripDetSummary.cc

```diff
@@ -1,47 +1,37 @@
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+
 #include "CondFormats/SiStripObjects/interface/SiStripDetSummary.h"
 
-void SiStripDetSummary::add(const DetId & detid, const float & value)
+void SiStripDetSummary::add(DetId detid, float value)
```

Comment:
Hi @pieterdavid 
Probably a naive question, but why are you removing the const'ness of the `DetId` at this stage?

Reply:
Hi @forthommel 
I changed them from "pass by const reference" to "pass by value" because that is simpler (I was changing the implementation of this method and my eye fell on it).
Both `DetId` and `float` are small (32 bits), so there is no expensive copy to avoid with `const &` (a reference should be at least as big, and needs to be dereferenced, which is actually more work). See http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rf-in for a longer explanation.

Reply:
Thanks a lot for your detailed explanation, @pieterdavid !